docs/001_develop/01_development-environment/002_launchpad/index.mdx

```diff
@@ -440,6 +432,10 @@ Then as soon as you start to type Genesis, the available commands are displayed
 | `Stop processes`      | Kills all running Genesis processes on the JVM or server and the WebUi. Use this command when processes are unresponsive. |
 | `Unpack project`      | Unpacks an available project generated in Genesis Create, and makes it active in the workspace. |
 
+:::warning
+`Unpack project` is deprecated and will be removed in the future.
```

Comment:
"is deprecated and no longer necessary, it is currently disabled by default but can be displayed back via settings".